ParkenDD/Colors.swift

```diff
@@ -90,4 +104,21 @@ extension UIColor {
 		}
 		self.init(red:red, green:green, blue:blue, alpha:alpha)
 	}
+
+    public var rgbaComponents: (red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat) {
```

Comment:
No need to set them as public. Only do so if the functions are part of a library.

Reply:
Removed; thank you.